salt/kube-apiserver/apiserver.jinja

```diff
@@ -51,6 +51,4 @@ KUBE_API_ARGS="--advertise-address={{ grains['ip4_interfaces']['eth0'][0] }} \
                --oidc-client-id=caasp-cli \
                --oidc-ca-file={{ pillar['ssl']['ca_file'] }} \
                --oidc-username-claim=email \
-               --oidc-groups-claim=groups \
-               --storage-backend=etcd2 \
```

Comment:
This option is configured in #line45 https://github.com/kubic-project/salt/pull/240/files#diff-7504d75a43d2c611b6f390b9c99a27d6L45